server/battleships/use_cases/implementation/src/test/java/eu/optas/use_cases/implementation/BoardD2BConverterTest.java

```diff
@@ -0,0 +1,36 @@
+package eu.optas.use_cases.implementation;
+
+import eu.optas.domain.Board;
+import eu.optas.domain.Ship;
+import eu.optas.use_cases.api.BoundaryBoard;
+import eu.optas.use_cases.api.BoundaryShip;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class BoardD2BConverterTest {
+
+    @Test
+    void convert() {
+        ShipD2BConverter shipD2BConverterMock = mock(ShipD2BConverter.class);
+        BoardD2BConverter boardD2BConverter = new BoardD2BConverter(shipD2BConverterMock);
+
+        List<List<Integer>> grid = new ArrayList<>();
+        grid.add(new ArrayList<>(Arrays.asList(0, 0, 0, 0)));
```

Comment:
new ArrayList<>() gali versti į List.of() (tik žinok, kad tas gautas list bus unmodifiable, tai jeigu bandysi modifikuoti - mes exception)